ui/StatusQ/src/StatusQ/Components/+qt6/StatusSoundEffect.qml

```diff
@@ -0,0 +1,32 @@
+import QtQuick 2.15
+import QtMultimedia 6.5
+
+import StatusQ 1.0
+
+Item {
+    id: root
+
+    property alias muted: soundEffect.muted
+    property alias volume: soundEffect.volume
+    property alias source: soundEffect.source
+    property alias playing: soundEffect.playing
+
+    property bool isError: soundEffect.status === soundEffect.Error
+    property string statusString: soundEffect.status
+
+    function play() {
+        soundEffect.play()
+    }
+
+    function stop() {
+        soundEffect.stop()
+    }
+
+    function convertVolume(volume) {
+        return AudioUtils.convertLogarithmicToLinearVolumeScale(volume);
```

Comment:
You can use https://doc.qt.io/qt-6/qtaudio.html#convertVolume

Reply:
isn't it used in the end? It's not directly exposed to qml, that's why we need the wrapper, right?

Reply:
It is exposed to QML, even in Qt6, albeit under a different namespace

Reply:
I've tried with the method used under this [namespace](https://doc.qt.io/qt-6/qtaudio.html) using the following implementation: 
``` 
function convertVolume(volume) {
        return QAudio.convertVolume(volume,
                                    QAudio.LogarithmicVolumeScale,
                                    QAudio.LinearVolumeScale)
    }
```
and the result is `../status-desktop/ui/StatusQ/src/StatusQ/Components/+qt6/StatusSoundEffect.qml:26: ReferenceError: QAudio is not defined`. I don't find now the source where I read that `convertVoume` is now private in qt6, so that's why I did the wrapper. Any other suggestion?

Reply:
The namespace is called `QtAudio`, not `QAudio` :) https://doc.qt.io/qt-6/qtaudio.html

Reply:
Same thing with `QtAudio`: `...status-desktop/ui/StatusQ/src/StatusQ/Components/+qt6/StatusSoundEffect.qml:26: ReferenceError: QtAudio is not defined`